src/odemis/odemisd/mdupdater.py

```diff
@@ -213,32 +221,34 @@ def updatePixelDensity(unused, lens=lens, comp=comp, binva=binva):
 
         # update pole position (if available), taking into account the binning
         if model.hasVA(lens, "polePosition"):
-            def updatePolePos(unused, lens=lens, comp=comp):
+            def updatePolePos(unused, lens=lens, comp_affected=comp_affected):
+                # "unused: because it might be magnification or binning"
```

Comment:
Well, here, it might be *polePosition* or binning.
BTW, could you also drop the quotes?
(Once this is updated, you can directly squash all the commits, and I'll merge the PR)